geoclue/geoclue/gc-provider.c

```diff
@@ -218,13 +236,13 @@ add_reference (GcIfaceGeoclue *geoclue,
 
 static void 
 remove_reference (GcIfaceGeoclue *geoclue,
-                  DBusGMethodInvocation *context)
+		  DBusGMethodInvocation *context)
```

Comment:
Looks like the automation made this and above slightly worse. Perhaps could remove from the commit.

Reply:
That looks ok in the code (2 tabs, 2 spaces for 18 character indentation). I think the problem is that gitlab's diff view uses 4 character tab width while the code is formatted with tab-width 8.